app/Filament/Resources/CourseResource.php

```diff
@@ -94,19 +93,4 @@ public static function getPages(): array
             'lessons.edit' => EditLesson::route('/{parent}/lessons/{record}/edit'),
         ];
     }
-
-    public static function getUrl(string $name = 'index', array $parameters = [], bool $isAbsolute = true, ?string $panel = null, ?Model $tenant = null): string
-    {
-        $parameters['tenant'] ??= ($tenant ?? Filament::getTenant());
-
-        $routeBaseName = static::getRouteBaseName(panel: $panel);
-        $routeFullName = "{$routeBaseName}.{$name}";
-        $routePath = Route::getRoutes()->getByName($routeFullName)->uri();
-
-        if (str($routePath)->contains('{parent}')) {
-            $parameters['parent'] ??= (request()->route('parent') ?? request()->input('parent'));
-        }
-
-        return route($routeFullName, $parameters, $isAbsolute);
-    }
```

Comment:
We don't need this piece of code, since we're passing `parent` as a parameter when calling `::getUrl()`.

Also, -`>uri()` at line 104 is causing an exeption as someone already reported in the Laravel Daily Blog

Reply:
With this one - you are right. We added this code to prevent some edge-cases with pagination and search, but it seems to work without this. So this will be implemented (hopefully nothing breaks)